packages/replay/src/canvas.ts

```diff
@@ -0,0 +1,54 @@
+import { getCanvasManager } from '@sentry-internal/rrweb';
+import type { Integration } from '@sentry/types';
+import type { ReplayConfiguration } from './types';
+
+interface ReplayCanvasOptions {
+  fps: number;
+  quality: number;
+}
+
+/** An integration to add canvas recording to replay. */
+export class ReplayCanvas implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'ReplayCanvas';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string;
+
+  private _canvasOptions: ReplayCanvasOptions;
+
+  public constructor() {
+    this.name = ReplayCanvas.id;
+    // TODO FN: Allow to configure this
+    // But since we haven't finalized how to configure this, this is predefined for now
+    // to avoid breaking changes
+    this._canvasOptions = {
+      fps: 4,
+      quality: 0.6,
+    };
```

Comment:
I'm going to update this and move the following over to this integration: https://github.com/getsentry/sentry-javascript/blob/develop/packages/replay/src/replay.ts#L348